docs/tutorials/tutorials.rst

```diff
@@ -49,6 +48,15 @@ Here you can find a collection of tutorials for various complexity ML workflows.
    workflows/03-dag-workflows/tutorial_2_advanced_workflow
    workflows/04_itwinai_argparser
  
- 
+
+HPO Workflows
```

Comment:
Maybe just Hyper-parameter optimization? "HPO" may not be clear to everyone

Reply:
Good point :)